### ISSUE TITLE
Shared textFont options

### ISSUE THREAD

**Issue by tylorr** [craftyjs/Crafty#350]
Any settings in textFont are shared across all text entities.

One fix is to move the decleration of `_textFont` into the `init()` function.

```
init: function () {
  this._textFont = {
    "type": "",
    "weight": "",
    "size": "",
    "family": ""
  };
  ...
```

I am not sure why this isn't also a problem for _text, it seems that your component extension code doesn't work when one of the properties is of type object.

This seems to be an example of this [issue](https://github.com/craftyjs/Crafty/issues/86)

**Comment by mpetrovic** [comment]
That's the only fix at the moment without rewriting how Crafty adds components to entities. Go ahead and write up a Pull Request if you got a sec.

**Comment by starwed** [comment]
This should be fixed now by #409, though I haven't explicitly tested it.  Closing, but if it turns out broken we can reopen.
